src/objective-c/examples/SwiftSample/ViewController.swift

```diff
@@ -71,7 +71,7 @@ class ViewController: UIViewController {
       NSLog("2. Response trailers: \(RPC.responseTrailers)")
     }
 
-    RPC.requestHeaders["My-Header"] = "My value"
+    RPC.requestHeaders.setObject("My value", forKey: "My-Header")
```

Comment:
Would you mind adding above this line (and above the other one you fixed too):
`TODO(jcanizales): Revert to using subscript syntax once XCode 8 is released.`